src/docs/guides/public-networking.md

```diff
@@ -140,6 +140,43 @@ The type of record to create is entirely dependent on your DNS provider.  Here a
 
 If your DNS provider doesn't support CNAME Flattening or dynamic ALIAS records, you can also change your domain's nameservers to point to Cloudflare's nameservers. This will allow you to use a CNAME record for the root domain. Follow the instructions listed on Cloudflare's documentation to <a href="https://developers.cloudflare.com/dns/zone-setups/full-setup/setup/" target="_blank">change your nameservers</a>.
 
+**Adding a root domain with www. subdomain to Cloudflare**
+
+If you want to add your root domain (e.g., `mydomain.com`) and the `www.` subdomain to Cloudflare and redirect all `www.` traffic to the root domain:
+
+1. Create a Custom Domain in Railway for your root domain (e.g., `mydomain.com`). Copy the `value` field. This will be in the form: `abc123.up.railway.app`.
+2. Add a `CNAME` DNS record to Cloudflare:
+    - `Name` → `@`.
+    - `Target` → the `value` field from Railway.
+    - `Proxy status` → `on`, should display an orange cloud.
+    - Note: Due to domain flattening, `Name` will automatically update to your root domain (e.g., `mydomain.com`).
+3. Add another `CNAME` DNS record to Cloudflare:
+    - `Name` → `www`.
+    - `Target` → `@`
+    - `Proxy status:` → on, should display an orange cloud.
+    - Note: Cloudflare will automatically change the `Target` value to your root domain.
+4. Enable Full SSL/TLS encryption in Cloudflare:
+    1. Go to your domain on Cloudflare.
```

Comment:
You switched from bullet points to numbers, please keep the bullet points throughout all sub-steps for consistency.